app/src/main/java/io/intrepid/contest/screens/entrysubmission/cropimage/CustomUCrop.java

```diff
@@ -0,0 +1,70 @@
+package io.intrepid.contest.screens.entrysubmission.cropimage;
+
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import com.yalantis.ucrop.BuildConfig;
+import com.yalantis.ucrop.UCrop;
+
+public class CustomUCrop {
+    private static final String EXTRA_PREFIX = BuildConfig.APPLICATION_ID;
+
+    private static final String EXTRA_INPUT_URI = EXTRA_PREFIX + ".InputUri";
+    private static final String EXTRA_OUTPUT_URI = EXTRA_PREFIX + ".OutputUri";
+
+    private static final String EXTRA_ASPECT_RATIO_X = EXTRA_PREFIX + ".AspectRatioX";
+    private static final String EXTRA_ASPECT_RATIO_Y = EXTRA_PREFIX + ".AspectRatioY";
+
+    private Intent cropIntent;
+    private Bundle cropOptionsBundle;
+
+    private CustomUCrop(@NonNull Uri source, @NonNull Uri destination) {
+        cropIntent = new Intent();
+        cropOptionsBundle = new Bundle();
+        cropOptionsBundle.putParcelable(EXTRA_INPUT_URI, source);
+        cropOptionsBundle.putParcelable(EXTRA_OUTPUT_URI, destination);
+    }
+
+    public static CustomUCrop of(String entryName, @NonNull Uri source, @NonNull Uri destination) {
+        CustomUCrop uCrop = new CustomUCrop(source, destination);
+        UCrop.Options options = new UCrop.Options();
+        options.setHideBottomControls(false);
+        options.setFreeStyleCropEnabled(false);
+        options.setToolbarTitle(entryName);
+        return uCrop
+                .withAspectRatio(1, 1).withOptions(options)
```

Comment:
formatting here is a little weird. we should be consistent. Probably should have each dont be on its own line.